### ISSUE TITLE
internal\certs\certs.go:955:30: nilness: tautological condition: nil == nil (govet)

### ISSUE THREAD

**Issue by atc0005** [atc0005/check-cert#817]
Linting error raised by govet:

```log
internal\certs\certs.go:955:30: nilness: tautological condition: nil == nil (govet)
                                        case md5RSASigVerifyErr == nil:
```

This was also flagged by a recent `gopls` release.
